sdk/search/azure-search/src/main/java/com/azure/search/util/SearchPagedFlux.java

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.search.util;
+
+import com.azure.core.util.paging.ContinuablePagedFlux;
+import com.azure.core.util.paging.ContinuablePagedFluxCore;
+import com.azure.core.util.paging.PageRetriever;
+import com.azure.search.SearchPagedResponse;
+import com.azure.search.models.SearchRequest;
+import com.azure.search.models.SearchResult;
+
+import java.util.function.Supplier;
+
+/**
+ * Implementation of {@link ContinuablePagedFlux} where the continuation token type is {@link SearchRequest}, the
+ * element type is {@link SearchResult}, and the page type is {@link SearchPagedResponse}.
+ */
+public final class SearchPagedFlux extends ContinuablePagedFluxCore<SearchRequest, SearchResult, SearchPagedResponse> {
+    /**
+     * Creates an instance of {@link SearchPagedFlux}.
+     *
+     * @param pageRetrieverProvider Supplied that handles retrieving {@link SearchPagedResponse SearchPagedResponses}.
+     */
+    public SearchPagedFlux(
+        Supplier<PageRetriever<SearchRequest, SearchPagedResponse>> pageRetrieverProvider) {
```

Comment:
The indentation is not right.

Reply:
Not sure how the indentation is wrong, I thought it was 4 spaces per level of indentation?